metron-platform/metron-elasticsearch/pom.xml

```diff
@@ -35,22 +35,28 @@
         </dependency>
         <dependency>
             <groupId>org.apache.metron</groupId>
-            <artifactId>metron-enrichment</artifactId>
```

Comment:
We should never have a dependency on `metron-enrichment`.  This causes all sorts of problems as a huge set of transitive dependencies get pulled in, and small dependency changes in seemingly unrelated places, can have a huge, ripple effect.  

Projects like `metron-enrichment` and `metron-elasticsearch` that include deployable artifacts are the "leafs" of our dependency tree and nothing else should depend on them.

Reply:
Rather than depending on `metron-enrichment`, we should just depend on the specific projects that we need.  This is much more direct and clear.